### PR TITLE
Add override for runAndWaitForPlanToComplete() with default timeout

### PR DESCRIPTION
**Pull request by Anmol-Singh-Jaggi**

### CODE REVIEW

File: subprojects/parseq-test-api/src/main/java/com/linkedin/parseq/ParSeqUnitTestHelper.java [hunk 1]
```diff
@@ -203,6 +203,18 @@ public ScheduledExecutorService getScheduler() {
     return runAndWaitForPlanToComplete("runAndWaitForPlanToComplete", task, time, timeUnit);
   }
 
+  public <T> T runAndWaitForPlanToComplete(Task<T> task) {
```
Comment (mchen07) [inline subprojects/parseq-test-api/src/main/java/com/linkedin/parseq/ParSeqUnitTestHelper.java:206]: Please add javadoc for all these pubic methods, also be consistent with any other javadoc format in this file.

Comment (Anmol-Singh-Jaggi) [reply subprojects/parseq-test-api/src/main/java/com/linkedin/parseq/ParSeqUnitTestHelper.java:210]: Added javadoc in a new commit.

**Comment by Anmol-Singh-Jaggi** [comment]
Can someone please review this?

**Comment by Anmol-Singh-Jaggi** [comment]
Done, however Github is not letting me resolve conflicts.
Can you do it please?

**Pull request merged**